src/main/java/testproject/TestProject.java

```diff
@@ -6,6 +6,7 @@ public static void main(String[] args) {
 		String alternativeName = "Max";
 		if (args.length > 0) {
 			System.out.println("Hello " + String.join(",", args) + "!");
+			System.out.println("How are you?");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.rz.is/coding_rules#rule_key=squid%3AS106)